Restructure of TLS Context (#46)
mulesoft/docs-anypoint-security: modules/ROOT/pages/index-secrets-manager.adoc

Anypoint Security secrets manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.

[NOTE]
====

> If you cannot Access Secrets Manager or you receive an error saying `Cannot Create Secret Groups`, you must add the correct permissions in access management.

Reviewer: ```suggestion
If you cannot access secrets manager or you receive an error saying `Cannot Create Secret Groups`, you must add the correct permissions in access management:
```

> * If you only need access to the secrets sanager and do not manage Anypoint Runtime Fabrics, add the user permissions under the *Access Management > Permissions > Secrets Manager* tab.

Reviewer: ```suggestion
* If you need access only to the secrets manager and do not manage instances of Anypoint Runtime Fabric, add the appropriate user permissions in the *Access Management > Permissions > Secrets Manager* tab.
```

> * If you manage Runtime Fabrics, you only need to add the *Manage Runtime Fabric* permission (you don't need to add the *Secrets Manager* permission separately). 

Reviewer: ```suggestion
* If you manage instances of Runtime Fabric, you need to add only the *Manage Runtime Fabric* permission (you don't need to add the *Secrets Manager* permission separately). 
```

Anypoint Security secrets manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.

[NOTE]

> If you don't see *Secrets Manager* in *Management Center* when you sign in to your Anypoint Platform account, contact your customer success manager to enable Secrets Manager for your account.

Reviewer 2: This is not correct. SM is available to all customers. There is no entitlement. The explanation is complicated:
Something like -- If you cannot Access Secrets Manager or you receive an error saying 'Cannot Create Secret Groups', it is necessary to get permissions added at Access Manager. This can be added in two ways:
--If you only need access to Secrets Manager and do not manage runtime fabrics, add permissions under the Access Manager->Secrets Manager tab.
--If you are also managing runtime fabrics, it is only necessary to have the "Manage Runtime Fabric" permission added. It is not necessary to also add Secrets Manager permissions separately.
